### PR TITLE
qe support

### PR DESCRIPTION
**Pull request by geonda**
This commit focuses on enhancing Quantum Espresso support, with most changes made to the class CalculationQE. Additionally, I have added  Python notebooks and output references to the tutorials folder. In terms of functionality only single-shot (static) and volume optimization calculations have been tested.

### CODE REVIEW

File: cluster_tools/fit_tool_qe.py
Comment (dimonaks) [inline cluster_tools/fit_tool_qe.py:1]: Please update description. There is still mention of VASP in it

File: siman/calc_manage.py [hunk 1]
```diff
@@ -1229,7 +1229,7 @@ def add_loop_finalize(u_scale_flag, fitted_v100_id):
             #by add_calculation; for u-ramping names are different
             cl = calc[it, setlist[0], 1]
 
-            calc[fitted_v100_id].path["output"] = cl.path["output"].replace('/1.', '/100.')
+            calc[fitted_v100_id].path["output"] = cl.path["output"].replace('./1.', './100.')
```
Comment (dimonaks) [inline siman/calc_manage.py:1232]: lets discus this change
calc[fitted_v100_id].path["output"] = cl.path["output"].replace('./1.', './100.')

File: siman/core/cluster_batch_script.py [hunk 2]
```diff
@@ -806,9 +806,13 @@ def write_footer(cl, set_mod = '', run_tool_flag = True,
 
 
 
-            run_command(cl, option = option, name = cl.id[0]+'.'+cl.id[1]+'.100'+name_mod+'.fitted', 
-                parrallel_run_command = parrallel_run_command, write = True, mpi = mpi, corenum = corenum, f = f)
-
+            # this is tmp solution 
+            if cl.calculator == 'qe':
```
Comment (dimonaks) [inline siman/core/cluster_batch_script.py:810]: Is there any difference here?

Comment (geonda) [reply siman/core/cluster_batch_script.py]: there were different naming for qe fitted output. fixed now

**Pull request merged**